tests/integration/modifiers/scoll-into-view-mock-test.js

```diff
@@ -0,0 +1,27 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, find } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import setupScrollIntoView from 'ember-scroll-modifiers/test-support/scroll-into-view-mock';
+
+module(
+  'Integration | Modifier | addon-test-support/scroll-into-view-mock',
+  function (hooks) {
+    setupRenderingTest(hooks);
+    setupScrollIntoView(hooks);
+    test('scroll into view returns correct parameters', async function (assert) {
```

Comment:
Add a newline above this one to separate the setup functions from the tests

Reply:
is this comment still valid?